### PR TITLE
Cares uri support

### PR DESCRIPTION
**Pull request by Prasek-prtes**
Added dns uri reply support. It's based on code of server reply.

### CODE REVIEW

File: src/lib/arec_parse_uri_reply.c [hunk 1]
```diff
@@ -0,0 +1,184 @@
+
```
Comment (gvanem) [inline src/lib/arec_parse_uri_reply.c:1]: The file-name is wrong; `ares_parse_uri_reply.c`.

File: src/tools/adig.c [hunk 2]
```diff
@@ -152,6 +155,7 @@ static const struct nv types[] = {
   { "DNSKEY",   T_DNSKEY },
   { "CAA",      T_CAA },
   { "ANY",      T_ANY }
+  { "URI",      T_URI }
```
Comment (gvanem) [inline src/tools/adig.c:158]: Add a `,` above that line. You probably have noted the AppVeyor jobs have failed.

**Comment by bradh352** [comment]
Seems like a number of build failures.  We'll also need a test case.

**Comment by bradh352** [comment]
moved to #411

**Pull request closed without merging**